Samples/StorageProviders/StorageProviders/packages.config

```diff
@@ -1,5 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Orleans.Core" version="1.0.10" targetFramework="net45" />
-  <package id="mongocsharpdriver" version="1.8.3" targetFramework="net45" />
+  <package id="Microsoft.Orleans.Core" version="1.2.0-beta" targetFramework="net452" />
```

Comment:
Samples are supposed to target latest stable release. So it should be either 1.1.3 or, better, 1.2.0. Although the change to 1.2.0 we'll need to hold until final 1.2.0 is out.